src/pyspiffe/config.py

```diff
@@ -8,17 +8,23 @@
 
 
 class Config:
-    """
-    Represents the configuration for a Workload API client
+    """Represents the configuration for a Workload API client
```

Comment:
nit: _The summary line should end with period, question or exclamation mark._ Both Summary lines in this file do not have any of those ending characters.

Reply:
good catch!